# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/magic/fix-install-paths.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/scripts/defs.mak.in	2025-08-20 01:08:38.807088978 +0200
+++ b/scripts/defs.mak.in	2025-08-20 01:10:13.652084441 +0200
@@ -33,13 +33,12 @@
 SCMDIR                 = ${libdir}/magic/scm
 TCLDIR                 = ${libdir}/magic/tcl
 
-# Install targets may be different if dist_prefix is set by configure
-INSTALL_BINDIR         = @DIST_DIR@/bin
-INSTALL_MANDIR         = @DIST_DIR@/share/man
-INSTALL_LIBDIR         = @DIST_DIR@/lib
-INSTALL_SYSDIR         = ${INSTALL_LIBDIR}/magic/sys
-INSTALL_SCMDIR         = ${INSTALL_LIBDIR}/magic/scm
-INSTALL_TCLDIR         = ${INSTALL_LIBDIR}/magic/tcl
+INSTALL_BINDIR         = ${BINDIR}
+INSTALL_MANDIR         = ${MANDIR}
+INSTALL_LIBDIR         = ${LIBDIR}
+INSTALL_SYSDIR         = ${SYSDIR}
+INSTALL_SCMDIR         = ${SCMDIR}
+INSTALL_TCLDIR         = ${TCLDIR}
 
 MAIN_EXTRA_LIBS	       = @extra_libs@
 LD_EXTRA_LIBS	       = @ld_extra_libs@
